### PR TITLE
PA-1150 Login page redesign and other small fixes

### PR DESCRIPTION
**Pull request by agalbrai**
image

I had some merge conflict trouble while doing a rebase, so hopefully things aren't out of wack.

### CODE REVIEW

File: frontend/src/features/projects/dispose/components/ExemptionRequest.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export default function ExemptionRequest({
             <TextArea
               label={'Rationale'}
               field={rationaleField}
-              className="col-md-5"
+              className="col-md-auto"
```
Comment (agalbrai) [inline frontend/src/features/projects/dispose/components/ExemptionRequest.tsx:67]: I noticed the notes field wasn't expanding, fixed it

File: frontend/src/features/projects/erp/forms/ExemptionEnhancedReferralCompleteForm.tsx [hunk 2]
```diff
@@ -53,6 +53,7 @@ const ExemptionEnhancedReferralCompleteForm = ({
         label="Exemption Rationale"
         field="exemptionRationale"
         disabled={true}
+        className="col-md-auto"
```
Comment (agalbrai) [inline frontend/src/features/projects/erp/forms/ExemptionEnhancedReferralCompleteForm.tsx:56]: Same here

File: frontend/src/layouts/PublicLayout.scss [hunk 3]
```diff
@@ -21,6 +21,23 @@
...
+  margin-bottom: 46px;
+
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+
+  /* for Firefox */
+  min-height: 0;
+
+  background-image: url('../assets/images/SeasideBlue.png');
```
Comment (Fosol) [inline frontend/src/layouts/PublicLayout.scss:36]: I don't see the `png` as an item in this PR?

Comment (Fosol) [reply frontend/src/layouts/PublicLayout.scss:36]: nvm, see it now

**Review by Fosol — APPROVED** [review]

File: frontend/src/layouts/PublicLayout.scss [hunk 4]
```diff
@@ -21,6 +21,23 @@
   min-height: 0;
 }
 
+//for displaying a background image on the login page
+.App-content-background {
```
Comment (devinleighsmith) [inline frontend/src/layouts/PublicLayout.scss:25]: why not put this on Login.scss?

Comment (devinleighsmith) [reply frontend/src/layouts/PublicLayout.scss:25]: The login form is the only place this background is going to be used right?

**Review by goemen — APPROVED** [review]
I like it

File: frontend/src/layouts/PublicLayout.scss (hunk 4 above)
Comment (agalbrai) [reply frontend/src/layouts/PublicLayout.scss:25]: I tried that, but App-content was the only place I could get the image to fit the entire screen.

Comment (devinleighsmith) [reply frontend/src/layouts/PublicLayout.scss:25]: you should be able to set the height of the .unauth class in Login.scss doing something like this:
```
@import '../../variables.scss';
height: calc(100vh - #{$footer-height} - #{$header-height} - #{$navbar-height});
```
you'll probably need to change the margin on the unauth class as well to padding.
That should make the .unauth element the same size as app-content

Comment (agalbrai) [reply frontend/src/layouts/PublicLayout.scss:25]: Got it!

File: frontend/src/features/account/Login.scss [hunk 5]
```diff
@@ -1,76 +1,91 @@
...
-    border-radius: 4px;
-  }
-  h1 {
-    padding-bottom: 18px;
-    font-family: 'BCSans', Fallback, sans-serif;
-    font-weight: 700;
+@import '../../variables.scss';
+
+.login {
+  height: calc(100vh - #{$footer-height} - #{$header-height});
```
Comment (devinleighsmith) [inline frontend/src/features/account/Login.scss:6]: Thanks for doing this.

**Pull request merged**